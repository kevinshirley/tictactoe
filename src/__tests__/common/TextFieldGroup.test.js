import React from 'react';
import { shallow } from 'enzyme';
import TextFieldGroup from '../../components/common/TextFieldGroup';

describe('TextFieldGroup', () => {
  it('should render correctly in "debug" mode', () => {
    const textFieldGroup = shallow(<TextFieldGroup debug />);
  
    expect(textFieldGroup).toMatchSnapshot();
  });
});