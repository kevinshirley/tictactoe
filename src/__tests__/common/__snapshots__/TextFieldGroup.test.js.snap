// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextFieldGroup should render correctly in "debug" mode 1`] = `
<div
  className="input-wrapper form-group"
>
  <div
    className="inner-wrap"
  >
    <label>
      <i
        className="material-icons"
      />
       
    </label>
    <div
      className="input-container"
    >
      <input
        className="form-control"
        type="text"
      />
    </div>
  </div>
</div>
`;
